corehq/apps/cloudcare/utils.py

```diff
@@ -17,12 +17,12 @@
 from corehq.util.quickcache import quickcache
 
 
-def can_user_access_web_app(app, user, domain):
+def can_user_access_web_app(user, app):
     """
-    :param app: app doc (not wrapped Application)
     :param user: either a WebUser or CommCareUser
-    :param domain: domain name
+    :param app: app doc (not wrapped Application)
     """
+    domain = app.get("domain")
```

Comment:
I think this should never be absent, and we probably want an error if it is?
```suggestion
    domain = app["domain"]
```

Reply:
Yeah that sounds right to me. Changed in 4a68aef15fcd1818f42fe3a0fb1d27ac567bf644